external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/KafkaSpout.java

```diff
@@ -58,27 +56,26 @@
 
 public class KafkaSpout<K, V> extends BaseRichSpout {
     private static final long serialVersionUID = 4151921085047987154L;
+    public static final long INITIAL_TIMER_DELAY_MS = 500;
```

Comment:
It isn't a default, but a hardcoded value. It's mostly here so it can be referred to from tests. I don't think it should be in Timer, since it's not a property inherent to Timer, but to how KafkaSpout uses Timer. I could rename it to INITIAL_COMMIT_AND_SUBSCRIPTION_REFRESH_DELAY_MS to make it clear what it's for?